modules/Campaigns/CaptchaValidate.php

```diff
@@ -34,33 +34,38 @@
  * In accordance with Section 7(b) of the GNU Affero General Public License version 3,
  * these Appropriate Legal Notices must retain the display of the "Powered by
  * SugarCRM" logo and "Supercharged by SuiteCRM" logo. If the display of the logos is not
- * reasonably feasible for  technical reasons, the Appropriate Legal Notices must
- * display the words  "Powered by SugarCRM" and "Supercharged by SuiteCRM".
- ********************************************************************************/
+ * reasonably feasible for technical reasons, the Appropriate Legal Notices must
+ * display the words "Powered by SugarCRM" and "Supercharged by SuiteCRM".
+ */
 
-/*********************************************************************************
+if (!defined('sugarEntry') || !sugarEntry) {
+    die('Not A Valid Entry Point');
+}
 
- * Portions created by SugarCRM are Copyright (C) SugarCRM, Inc.
- * All Rights Reserved.
- * Contributor(s): ______________________________________..
- ********************************************************************************/
+$log = LoggerManager::getLogger();
 
- /**Check captcha validation here.
- *
- */
-require_once('include/recaptcha/recaptchalib.php');
-
-$admin=new Administration();
+$admin = new Administration();
 $admin->retrieveSettings('captcha');
-if($admin->settings['captcha_on']=='1' && !empty($admin->settings['captcha_private_key'])){
-	$privatekey = $admin->settings['captcha_private_key'];
-}else
-	die("Captcha settings not found");
-$response = recaptcha_check_answer($privatekey,
-									$_SERVER["REMOTE_ADDR"],
-									$_REQUEST["recaptcha_challenge_field"],
-									$_REQUEST["recaptcha_response_field"]);
-if(!$response->is_valid){
-	die("Invalid captcha entry, go back and fix. ". $response->error. " ");
+
+if (
+    $admin->settings['captcha_on'] == '1'
+    && !empty($admin->settings['captcha_private_key'])
+) {
+    $privatekey = $admin->settings['captcha_private_key'];
+} else {
+    $error = 'Captcha settings not found';
+    $log->fatal($error);
+    die($error);
+}
+
+$recaptcha = new \ReCaptcha\ReCaptcha($privatekey);
```

Comment:
this piece of code seems code duplication of https://github.com/salesagility/SuiteCRM/pull/5531/files#diff-e10598f4b5eca1e28a6b573067a57547R69

Reply:
I have added a util file to remedy the issue (include/utils/recaptcha_utils.php).